src/components/exit-this-page/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Exit this page
+description: Give users a way to quickly exit a service, website or application quickly and safely.
+section: Components
+aliases:
+backlog_issue_id: 213
+layout: layout-pane.njk
+status: Experimental
+statusMessage: This component is currently experimental and part of a community consultation.
+---
+
+Give users a way to quickly exit a service, website or application quickly and safely.
+
+For service journeys, this component must be used together with the pattern to help a user [exit the page quickly](/patterns/exit-the-page-quickly/).
+
+## When to use this component
+
+Use the component on pages with sensitive information that could:
+- put someone at risk of abuse or retaliation
+- reveal someone’s plans to avoid or escape from harm
+
+For example, when a potential victim is using a service to help them leave a domestic abuser.
+
+You can use this component on either:
+- all the pages in a service
+- parts of the journey with sensitive information
+
+You can also use this component for standalone content pages, such as dashboards and guidance.
+
+## When not to use this component
+
+Do not use this component if the service or content is unlikely to put a user at risk. See the [Exit the page quickly](/patterns/exit-the-page-quickly/) pattern for examples of at-risk and sensitive topics.
+
+Keep in mind that seeing this component might discourage certain users from using your service. If the user does not identify themselves as being at risk, they might see the button on a service and decide it’s not relevant to them.
+
+## How it works
+
+There are 2 ways to use the exit this page component. You can use HTML or, if you are using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
+
+The Exit this page component has several parts, including:
+- a button, that takes the user to another website
+- a secondary link, to give people that use assistive technologies a way to activate the component
+- a loading overlay, to immediately clears any content off the browser until the next website loads
+
+All parts of this component must be used together and alongside the Exit the page quickly pattern.
+
+## Adding a button
+
+Position the button component immediately below the header of your page. Its position should be 'sticky', so that as the user scrolls down the page, the button remains visible and is easy to activate. 
+
+On mobile devices, position the component at the bottom of the screen at all times. This makes it easier to press with their thumb and hidden with their hand.
```

Comment:
I think this is old information. 

Although this was the original desire, following [the spike into positioning](https://github.com/alphagov/govuk-frontend/pull/2947), we found that having the button at the bottom of the screen caused it to be obscured by the on-screen keyboard when it was open. 

We've since kept it so that it's initially placed near the top of the page (below the header, but before the H1) and sticks to the top of the screen when scrolling down.

Reply:
```suggestion
On mobile devices, position the button immediately below the header and make it 'stick' to the top of the screen when the user scrolls down the page.
```